README.md

```diff
@@ -26,15 +26,27 @@ pixabay.authenticate('username', 'api_key');
 // Set default query parameters to make with every request
 pixabay.defaults({safesearch: 'true'});
 
-// Create a request promise
-var requestPromise = pixabay
+// Create a request handler
+var handler = pixabay
     .request()
     .query({order: 'latest'})
-    .search(['dogs', 'puppies'])
-    .get();
+    .search(['dogs', 'puppies']);
+
+// Get the request promise
+var request = handler.get();
```

Comment:
There's a problem with that. `get` returns a promise of the request, not the handler, and the handler is needed to call `next`.

I'm open to ideas on how to better design it.